test/integration/connection-monitoring-and-pooling/multiple_db.test.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const { setupDatabase } = require('../shared');
+
+describe('Multiple Databases', function () {
+  before(function () {
+    return setupDatabase(this.configuration, ['integration_tests2']);
+  });
+
+  it('should not leak listeners', function (done) {
+    var configuration = this.configuration;
+    const client = configuration.newClient({}, { sslValidate: false });
+    client.connect(function (err, client) {
+      for (var i = 0; i < 100; i++) {
+        client.db('test');
+      }
+
+      client.close(done);
+    });
+  });
```

Comment:
I think this test can be deleted, no idea what listeners its referring to here

Reply:
Yeah, the multiple_db test should have kept the first test "shouldCorrectlyUseSameConnectionsForTwoDifferentDbs" (maybe update the name) per my original comment and deleted the other two